backend/src/main/java/ca/gov/dtsstn/cdcp/api/data/repository/ConfirmationCodeRepository.java

```diff
@@ -0,0 +1,13 @@
+package ca.gov.dtsstn.cdcp.api.data.repository;
+import java.time.Instant;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import ca.gov.dtsstn.cdcp.api.data.entity.ConfirmationCodeEntity;
+
+
+@Repository
+public interface ConfirmationCodeRepository extends JpaRepository<ConfirmationCodeEntity, String> {
+
+	void deleteByExpiryDateLessThan(Instant currentInstant);
```

Comment:
I think it would be beneficial to return a count of the number of deleted codes, to be used for metrics and auditing purposes in a future PR.

Reply:
Also, I don't think `currentInstant` is a great name.. it implies that you have to pass in the current time, which isn't the case.